src/js/components/Ballot/PositionItem.jsx

```diff
@@ -86,9 +87,8 @@ export default class PositionItem extends Component {
       {/* One Position on this Candidate */}
         <Link to={speakerLink}>
           { position.speaker_image_url_https ?
-            <img className="img-square position-item__avatar"
-                  src={position.speaker_image_url_https}
-                  width="50px"
+            <ImageHandler className="img-square position-item__avatar"
+                  imageUrl={position.speaker_image_url_https}
```

Comment:
Hi @zachmonteith, it seems like it would be fine to force an image width with a prop passed into ImageHandler (like the original img tag had).